<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>715</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>715</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gear_resources.qrc">:/bgr.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>571</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QPushButton" name="zoomIn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>201</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Увеличить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zoomOut">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Уменьшить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restoreView">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Исходный вид</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tooth">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>200</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Кривые/Зубья</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toggleWireframe">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>240</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Сетка модели (вкл/выкл)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toggleBgWireframe">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>280</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Сетка фона (вкл/выкл)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zoomTooth">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>350</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Увеличить зуб</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gear_resources.qrc"/>
 </resources>
 <connections/>
</ui>
